llvm/lib/Target/RISCV/RISCVRegisterInfo.td

```diff
@@ -549,7 +549,7 @@ def VRM8 : VReg<VM8VTs, (add V8M8, V16M8, V24M8, V0M8), 8>;
 
 def VRM8NoV0 : VReg<VM8VTs, (sub VRM8, V0M8), 8>;
 
-def VMV0 : VReg<VMaskVTs, (add V0), 1>;
+def VMV0 : VReg<VMaskVTs, (add V0, V0, V0, V0, V0), 1>;
```

Comment:
What's going on here? Are you trying to define a register class with a single physical register, repeated multiple times?

Reply:
Aha, it's a mistake. I will remove this change.